docs/open-source-community/developer/backend/coding-style.md

```diff
@@ -0,0 +1,62 @@
+# Coding Style
+
+## Basics
+
+1. Avoid cryptic abbreviations. Single letter variable names are fine in very short methods with few variables, otherwise, make them informative.
```

Comment:
I did, in line 2 next section..

I'll do another round of clean up again to simplify this further (less instructions the better), since some of them like 120 line length are auto-enforced and not worth mentioning.

Reply:
PS: Planning to add how to build and run backend into tis directory